src/xrpld/core/Config.h

```diff
@@ -78,8 +78,15 @@ struct FeeSetup
     XRPAmount account_reserve{10 * DROPS_PER_XRP};
 
     /** The per-owned item reserve requirement in drops. */
+    // equivalent of 0.2 * DROPS_PER_XRP but with integer math
     XRPAmount owner_reserve{2 * DROPS_PER_XRP};
```

Comment:
How is this `equivalent of 0.2 * DROPS_PER_XRP`?

Reply:
Forgot to change that back - I couldn't get typing to be happy with 1/0.2 so I went back to 10/2